### DISCUSSION TITLE
Add multiple different locations as training data

### DISCUSSION THREAD

**Discussion by ogamingSCV** [imartinez/privateGPT#731] [Ideas]
I think it would be cool to be able to add multiple locations as a variable for the training data. This way I would be able to just add paths to my stored documents, notes etc. Maybe even a remote source, like from a NAS serve. 

With this it might be possible to start a new ingrest every night when files have changed and I am always able to access my newest documents. 

Also good idea is to skip fails during ingrest and only showing a list of failed files at the end (already a pull request available).

**Comment by sime2408** [comment]
You gave me great ideas :). I am planning to create a feature for Chroma over IP and for documents NAS as you said with CRON schedulers to pick new ones up. For now, in my [fork](https://github.com/sime2408/scrapalot-chat), I created sub-directories and Chroma collections to granulate or separate multiple databases. So you have max 2 levels of sources: the parent name of the database and the sub-category stored as Chroma collection/tag. Examples of database `psychology`: 
```
psychology
-  parenting
-  ...
```
